react/ProductList.tsx

```diff
@@ -7,8 +7,8 @@ import { AVAILABLE } from './constants/Availability'
 interface Props {
   items: Item[]
   loading: boolean
-  onQuantityChange: (uniqueId: string, value: number) => void
-  onRemove: (uniqueId: string) => void
+  onQuantityChange: (uniqueId: string, value: number, item?: Item) => void
```

Comment:
This bugs me a bit because the `item` argument makes `uniqueId` redundant. I thought about removing `uniqueId` and making `item` required, but it doesn't make sense to require the whole Item object just to change its quantity, so I guess this should be fine...

Reply:
I thought about that too! It is strange to pass the same thing twice, but removing `uniqueId` would also be a breaking change to the other components using ProductList.
Also, I agree that it should not be required to pass the whole item just to update the quantity. This is kind of a specific use-case since the minicart needs to send events to the Pixel Manager and it needs to know almost everything about the item